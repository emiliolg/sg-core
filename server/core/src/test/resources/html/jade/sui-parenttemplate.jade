doctype html
html(sg-view="tekgenesis.service.html.Views")
    head
        title= 'Parent Template'
    body
        h1= 'Include'
        include ./sui-subtemplate