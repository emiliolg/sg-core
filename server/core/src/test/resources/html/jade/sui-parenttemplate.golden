<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views">
    <head>
      <title>Parent Template</title>
    </head>
    <body>
      <h1>Include</h1>
      <h2>Child Template</h2>
    </body>
</html>